media/js/mozorg/home/fundraiser-eoy2018.js

```diff
@@ -0,0 +1,33 @@
+(function($, Mozilla) {
+    'use strict';
+
+    var $fundraiser = $('#fundraiser');
```

Comment:
Nit: not a blocker, but there's a couple of lines of jQuery here that could pretty easily be vanilla JS with just a couple of small changes :)